components/config-provider/index.tsx

```diff
@@ -327,6 +331,18 @@ const ConfigProvider: React.FC<ConfigProviderProps> & {
 ConfigProvider.ConfigContext = ConfigContext;
 ConfigProvider.SizeContext = SizeContext;
 ConfigProvider.config = setGlobalConfig;
+ConfigProvider.useConfig = useConfig;
+
+Object.defineProperty(ConfigProvider, 'SizeContext', {
+  get: () => {
+    warning(
+      false,
+      'ConfigProvider',
+      'ConfigProvider.SizeContext is deprecated. Please use `ConfigProvider.useConfig().componentSize` instead',
+    );
+    return ConfigProvider.SizeContext;
```

Comment:
dead loop here. Just return `SizeContext` is enough.

Reply:
fixed, sorry to ignore this 😥